shared/src/components/BadgeAttachment.scss

```
@@ -7,7 +7,7 @@
         z-index: 1;
         border: none;
         border-radius: 1rem;
-        background-color: rgba(0, 0, 0, 0.8);
+        background-color: transparentize($body-bg-color-dark, 0.2);
```

Suggestion: I would consider this easier to reason about, as you don't need to guess what the `transparentize()` function does (does it make it 20% more transparent, or set transparency to 20%?)
```suggestion
        background-color: rgba($body-bg-color-dark, 0.8);
```

---

Updated.